src/Components/Alert/Alert_Color_Options.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Alert\Components\Alert;
+
+class Alert_Color_Options implements Color_Options_Manager {
+
+	/**
+	 * @var array{name: string, class: string}
```

Comment:
```suggestion
	 * @var array<string, array{name: string, class: string}>
```
I finally found the correct phpstan definition for this.